docker/generate-vectortiles/export-list.sh

```diff
@@ -16,7 +16,7 @@ function export_local_mbtiles() {
        exit 500
     fi
 
-    filter_deprecation tilelive-copy \
```

Comment:
why remove filter_deprecation? Is it because they no longer show up?

Reply:
OK, it works with `filter_deprecation`. I will return it.